sendtweet

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
I suggest to add a link to the original source of this script. The license is compatible with ours, so fine to include.